core/src/components/menu/test/safe-area/menu.e2e.ts

```diff
@@ -0,0 +1,117 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+/**
+ * This behavior does not vary across modes.
+ */
+configs({ modes: ['md'] }).forEach(({ title, config, screenshot }) => {
+  test.describe(title('menu: safe area'), () => {
+    test.beforeEach(async ({ page }) => {
+      await page.goto(`/src/components/menu/test/safe-area`, config);
+    });
+    test.describe('side: start', () => {
+      test('should render with safe area when notch is on the left', async ({ page }) => {
+        const ionDidOpen = await page.spyOnEvent('ionDidOpen');
+        const ionDidClose = await page.spyOnEvent('ionDidClose');
+
+        await page.evaluate(() => {
+          const style = document.createElement('style');
+          style.innerHTML = `
+            :root {
+              --ion-safe-area-left: 50px;
+              --ion-safe-area-right: 10px;
+            }
+          `;
+          document.head.appendChild(style);
+        });
+
+        await page.click('#open-start');
+        await ionDidOpen.next();
+
+        await expect(page).toHaveScreenshot(screenshot(`menu-start-safe-area-left-notch`));
+
+        await page.locator('[menu-id="start-menu"]').evaluate(async (el: HTMLIonMenuElement) => {
+          await el.close();
+        });
+        await ionDidClose.next();
```

Comment:
Do we need to close the menus? It adds extra time to the tests, and I think the page will be refreshed at the start of each new test anyway due to the `beforeEach`.